starboard/shared/win32/file_internal.cc

```diff
@@ -25,7 +25,7 @@
 #include "starboard/shared/win32/error_utils.h"
 #include "starboard/shared/win32/wchar_utils.h"
 
-#define O_ACCMODE O_RDWR | O_WRONLY | O_RDONLY
+#define O_ACCMODE (O_RDONLY | O_WRONLY | O_RDWR)
```

Comment:
Does the flags order make any difference?

Reply:
No but I think the missing bracket was causing some issue in Windows. We don't ship SB17 windows anymore but it'd still be good to correct this.